tools/ratbagd.py

```diff
@@ -347,6 +347,11 @@ def name(self):
         """The device name, usually provided by the kernel."""
         return self._get_dbus_property("Name")
 
+    @GObject.Property
+    def device_type(self):
+        """The type of this device: unspecified(0), mouse(1), keyboard(2) or other(3)"""
+        return self._get_dbus_property("DeviceType")
+
```

Comment:
I think it's probably a good idea to add the enums in ratbagd somehow. Maybe like [here](https://github.com/libratbag/libratbag/blob/ceaa3c3ad5c7b4b7ebe1bc5ca8ae1c1a30ecb006/tools/ratbagd.py#L37) (just an idea).